docs/source/markdown/podman-history.1.md

```diff
@@ -28,12 +34,7 @@ Valid placeholders for the Go template are listed below:
 | .CreatedSince   | Elapsed time since the image layer was created       |
 | .Size           | Size of layer on disk                                |
 | .Comment        | Comment for the layer                                |
-## OPTIONS
-
-Print the numeric IDs only (default *false*).
-#### **--format**=*format*
-
-Alter the output for a format like 'json' or a Go template.
+| .Tags           | Image tags |
```

Comment:
I snuck this in because I didn't think my description would be too off the mark. If it is, lmk and I'll delete it.